### PR TITLE
#2091 remove css rule which causes has-addons to not work correctly

### PR DESCRIPTION
**Pull request by gvdp**
This is a **bugfix**.

### Proposed solution
Remove css line which causes extra margin mentioned in #2091 

### Tradeoffs
Removed css rule will not be used anywhere else so removing it will not cause issues.

### Testing Done

Icons now look like this:

image

With following html:

```
<div class="tags has-addons is-right">
  <span class="tag">Package</span>
  <span class="tag is-primary">Bulma</span>
</div>
```

### CODE REVIEW

File: sass/elements/tag.sass [hunk 1]
```diff
@@ -34,7 +34,7 @@ $tag-delete-margin: 1px !default
     justify-content: flex-end
     .tag
       &:not(:first-child)
-        margin-left: 0.5rem
+        margin-left: 0
```
Comment (jgthms) [inline sass/elements/tag.sass]: You're targeting all tags, not the addons one only. You need to put the the code under `&.has-addons`.

**Comment by gvdp** [comment]
Yep, you're right. I made the changes in the css and then didn't copy it in the right place in the .sass, my bad. Tested some more combinations now with the actual compiled css, looks like this so I think all still works:

image

**Comment by jgthms** [comment]
You need to remove the compiled CSS from your PR.

**Comment by gvdp** [comment]
> You need to remove the compiled CSS from your PR.

Done

**Pull request merged**